database/migrations/2022_11_17_171458_remove_date_created.php

```diff
@@ -17,7 +17,7 @@ public function up()
             $table->dropColumn('date_reservation');
         });
     }
-
+    
```

Comment:
its just better to always discard some unwanted changes  like this, it does not matter in small project like this, but in bigger projects and configuration files where lot of devs create these unwanted changes, there might be merge conflicts in the future